src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const nameContent = `My name is ${person.name}`;
+
+  let partnerContent;
+
+  if (person.isMarried) {
+    partnerContent = person.sex === 'f'
+      ? `${person.partnerName} is my husband`
+      : `${person.partnerName} is my wife`;
+  } else {
+    partnerContent = 'I am not married';
+  }
```

Comment:
don't complicate your code

you can just do this check:
const partner = sex === 'm' ? 'wife' : 'husband';

and then in the `return` write this condition:

```
<p className="Person__partner">
  {isMarried
    ? `${partnerName} is my ${partner}`
     : 'I am not married'
    }
</p>
```